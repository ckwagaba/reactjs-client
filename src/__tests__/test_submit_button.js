import React from 'react';
import { shallow } from 'enzyme';
import SubmitButton from '../components/submit_button';

describe ('SubmitButton component', () => {

  it('should exist', () => {
    const component = <SubmitButton />;
    expect(component).toMatchSnapshot();
  });

});
