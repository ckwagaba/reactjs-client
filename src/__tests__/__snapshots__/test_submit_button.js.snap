// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton component should exist 1`] = `<SubmitButton />`;
